husarion: #namespace

    running_step: 0.04 # amount of time the control will be executed
    
    #qlearn parameters
    alpha: 0.1
    gamma: 0.7
    epsilon: 0.9
    epsilon_discount: 0.999
    nepisodes: 20000
    nsteps: 2000

    n_actions: 9 # Forwards,TurnLeft,TurnRight

    init_linear_forward_speed: 0.0 # Initial linear speed in which we start each episode
    init_linear_turn_speed: 0.0 # Initial angular speed in shich we start each episode
    precision: 1 # Number of decimals that we will accept in all sensor readings

    linear_forward_speed: 0.3 # Speed for going fowards
    linear_turn_speed: 0.0 # Lienare speed when turning
    angular_speed: 0.4 # Angular speed when turning Left or Right
    move_base_precision: 0.2 # We allow this difference between the asked linear speed and the real one. Note that angular is not because controller is too imprecise in sim.
    
    new_ranges: 20 # How many laser readings we jump in each observation reading, the bigger the less laser resolution
    max_laser_value: 5.0 # Value considered Ok, no wall
    min_laser_value: 0.05 # Value considered there is an obstacle or crashed
    
    

